.travis.yml

```diff
@@ -1,11 +1,17 @@
 language: node_js
-sudo: false
-node_js:
-  - 0.10
+dist: trusty
+sudo: required
+node_js: 6
+env:
+  - PATH=$HOME/purescript:$PATH
 install:
+  - TAG=$(wget -q -O - https://github.com/purescript/purescript/releases/latest --server-response --max-redirect 0 2>&1 | sed -n -e 's/.*Location:.*tag\///p')
+  - wget -O $HOME/purescript.tar.gz https://github.com/purescript/purescript/releases/download/$TAG/linux64.tar.gz
```

Comment:
Ok, do you have an example where this is used?

BTW I did it like this because that seems to be the standard (like everywhere in purescript org)